src/consts/interfaces.consts.ts

```diff
@@ -167,6 +167,10 @@ export interface IPrepareDepositTxsResponse {
 	unsignedTxs: IPeanutUnsignedTransaction[]
 }
 
+export interface IPrepareDepositFulfillmentTxsResponse {
```

Comment:
depositfulfillment? what does  this mean?

please rename. I suggest IPrepareRequestFulfillmentTxsResponse

Reply:
like in IPrepareXchainRequestFulfillmentTransactionProps {